service/src/main/java/bio/terra/externalcreds/ExternalCredsSpringConfig.java

```diff
@@ -19,6 +21,8 @@
 @EnableRetry
 @EnableTransactionManagement
 @EnableConfigurationProperties
+@EnableCaching
+@EnableScheduling
```

Comment:
`EnableCaching` was missing so the existing cache we had didn't even work
`EnableScheduling` moved from `ExternalCredsCronApplication` because all instances need scheduling now to handle cache resets.